cdc/processor.go

```diff
@@ -811,16 +811,17 @@ func (p *oldProcessor) addTable(ctx context.Context, tableID int64, replicaInfo
 
 	startPuller := func(tableID model.TableID, pResolvedTs *uint64, pCheckpointTs *uint64) sink.Sink {
 		// start table puller
-		enableOldValue := p.changefeed.Config.EnableOldValue
-		span := regionspan.GetTableSpan(tableID, enableOldValue)
+		span := regionspan.GetTableSpan(tableID)
 		kvStorage, err := util.KVStorageFromCtx(ctx)
 		if err != nil {
 			p.sendError(err)
 			return nil
 		}
+		// NOTICE: always pull the old value internally
+		// See also: TODO(hi-rustin): add issue link here.
```

Comment:
What is the issue link?

Reply:
Issue has not been created yet, I will add it later after I create a issue after summarizing.

Reply:
💪